src/components/EpicButtons.test.tsx

```diff
@@ -5,47 +5,41 @@
 import '@testing-library/jest-dom';
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import testData from './ContributionsEpic.testData';
+import { factories } from '../factories';
 
 import { EpicButtons } from './EpicButtons';
 
 describe('EpicButtons', () => {
     it('Renders the primary button', () => {
-        const variant = {
-            ...testData.content,
-            name: 'Example',
-            showTicker: false,
+        const variant = factories.variant.build({
             cta: {
                 text: 'Button text!',
-                baseUrl: 'https://support.theguardian.com/support',
             },
-        };
-        const tracking = testData.tracking;
+        });
+        const tracking = factories.tracking.build();
 
         render(<EpicButtons variant={variant} tracking={tracking} />);
 
         expect(screen.getByText('Button text!')).toBeInTheDocument();
     });
 
     it('Renders nothing when no primary CTA', () => {
-        const variant = {
-            ...testData.content,
-            name: 'Example',
-            showTicker: false,
+        const variant = factories.variant.build({
             cta: undefined,
-        };
-        const tracking = testData.tracking;
+            secondaryCta: {
+                text: 'Secondary button!',
+                baseUrl: 'https://support.theguardian.com/support',
+            },
+        });
+        const tracking = factories.tracking.build();
 
         const { queryByTestId } = render(<EpicButtons variant={variant} tracking={tracking} />);
 
         expect(queryByTestId('epic-buttons')).toBeNull();
```

Comment:
Unrelated to this PR, but noticed this and couldn't figure it out at first. Better might be to simply check that the render result is null/empty here?

Reply:
Yeah it's a good point, and that's exactly how I initially wrote this test. However it seems the way `render` works here (I guess via jest-dom) is that it's rendered within a minimal HTML doc, so the result of rendering isn't just null because you still get the outer html doc.